core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -424,77 +576,76 @@ public long requestStartTimeMillis() {
 
     @Override
     public long requestStartTimeNanos() {
-        ensureAvailability(REQUEST_START);
+        ensureAvailable(RequestLogProperty.REQUEST_START_TIME);
         return requestStartTimeNanos;
     }
 
     @Override
-    public long requestFirstBytesTransferredTimeNanos() {
-        ensureAvailability(REQUEST_FIRST_BYTES_TRANSFERRED);
-        return requestFirstBytesTransferredTimeNanos;
+    public Long requestFirstBytesTransferredTimeNanos() {
+        ensureAvailable(RequestLogProperty.REQUEST_FIRST_BYTES_TRANSFERRED_TIME);
+        return requestFirstBytesTransferredTimeNanosSet ? requestFirstBytesTransferredTimeNanos : null;
```

Comment:
I think we can just return `requestFirstBytesTransferredTimeNanos`.

Reply:
No. A request can be complete without sending anything to the wire. In that case, `requestFirstBytesTransferredTimeNanosSet` is `false` and `null` has to be returned.